tests/python/gaia-ui-tests/gaiatest/apps/settings/app.py

```diff
@@ -9,7 +9,7 @@
 
 class Settings(Base):
 
-    name = 'Settings'
+    origin = 'app://settings.gaiamobile.org'
```

Comment:
Perhaps it would be better to just add the origin stuff for now and then later remove the name thing?